.gitignore

```diff
@@ -9,3 +9,4 @@
 /doc/
 /public/finder-frontend/
 /coverage/*
+app/assets/javascripts/templates.js
```

Comment:
I think this file needs to exist but be blank. I just tried deleting it and restarting my local server and I got an error. Can you check?

Reply:
I've just checked - yes, if you delete the file, it causes an error. But `app/assets/javascripts/templates.js` exists in the repo already, this commit will just ignore any changes to it, which is what we want 😄 people who newly install the repo will continue to get a blank `templates.js` file by default.

Reply:
Okay that's fine, but I think this should be explained somewhere. Otherwise at some point someone is going to notice that this file is ignored but still in the repo and try to delete it.

Reply:
Have added a comment to the `.gitignore`.

Reply:
EDIT: git still tries to commit changes to this file 😓 so have removed from .gitignore.